app/pages/lab-fem/components/fem-multi-image-subject-options-editor.jsx

```diff
@@ -55,9 +63,58 @@ export default function FemMultiImageSubjectLayoutEditor ({
             checked={enableSwitchingChecked}
             onChange={toggleEnableSwitching}
           />
-          Allow Separate Frames View - <small>volunteers can choose flipbook or a one-column separate frames view</small>
+          Allow Separate Frames View - <small>volunteers can choose flipbook or a separate frames view</small>
```

Comment:
A small thing but labels should be short. I'd put the supplementary text outside the label, to make the label more accessible.